### DISCUSSION TITLE
请明确一下，密码到底用哪一个？

### DISCUSSION THREAD

**Discussion by sherlocknieh** [lxysl/Auto_Attendance#27] [Q&A] [answered]
A.[新版统一身份认证](https://ca.csu.edu.cn/authserver)的密码
B.[旧版统一身份认证](https://ca.its.csu.edu.cn)的密码（默认是身份证号后六位）
C.绑定微信信网中心时用的密码（身份证号）

**Comment by lxysl** [comment]
你好，请看最新的issue

**Comment by sherlocknieh** [comment, marked as answer]
确定了，是 A.**新版**统一身份认证的密码，已打卡成功
